packages/mobile/src/send/SendAmount.tsx

```diff
@@ -202,9 +209,26 @@ function SendAmount(props: Props) {
     }
   }, [props.route, localCurrencyCode, localCurrencyExchangeRate, dollarAmount])
 
+  const onReviewButtonPressed = async () => {
+    setReviewButtonToPressed(true)
+    // Wait until verification status is known to proceed
+    while (recipientVerificationStatus === RecipientVerificationStatus.UNKNOWN) {
+      await wait(250)
+    }
+
+    setIsFetchingAddressMapping(false)
```

Comment:
Without the context of the PR, I don't know if I would have realized that fetchingAddressMapping is equivalent to getting the recipientVerificationStatus

Reply:
Makes sense. Renamed to `fetchingValidationStatus` to make that more clear